docs/index.md

```diff
@@ -1043,7 +1041,7 @@ Allows use of third-party reporters. For example, [mocha-lcov-reporter](https://
 
 ### `--reporter-option <option>, -O <option>, --reporter-options <option>`
 
-> *Updated in v6.0.0.  Can be specified multiple times.  `--reporter-options` is now an alias for `--reporter-option`.*
+> _Updated in v6.0.0. Can be specified multiple times. `--reporter-options` is now an alias for `--reporter-option`._
```

Comment:
While the options are very related, this is **incorrect**.
`--reporter-option` is _only_ for a _single_ option; `--reporter-options` handles a comma-separated list of options.

Reply:
I'm not sure where you're getting that information.  `reporter-options` is an alias of `reporter-option`.

Reply:
It was meant to be the replacement of `--reporter-options`, **never** an alias.

https://github.com/mochajs/mocha/pull/3556#discussion_r231503173
#3564

Reply:
ah, right.  https://github.com/mochajs/mocha/issues/3564#issuecomment-458740928